### PR TITLE
User can view their entry

### PR DESCRIPTION
**Pull request by moamel97**
Kapture 2021-10-28 at 20 25 40

### CODE REVIEW

**Comment by moamel97** [comment]
Kapture 2021-10-30 at 16 59 01

File: css/styles.css [hunk 1]
```diff
@@ -1,3 +1,8 @@
+
+/* .padding-top {
+  padding-top: 26px;
+} */
```
Comment (codingcodymiller) [inline css/styles.css:4]: Please remove commented code prior to submitting for review

File: css/styles.css [hunk 2]
```diff
@@ -58,12 +68,34 @@ label {
   line-height: 22px;
 }
 
+.position {
+  position: relative;
+  left: 20rem;
```
Comment (codingcodymiller) [inline css/styles.css:73]: This is a _**huge**_ value. Why is this necessary?

File: js/main.js [hunk 3]
```diff
@@ -24,5 +24,98 @@ function submitForm(event) {
...
+}
+window.addEventListener('DOMContentLoaded', handleDomLoaded);
+
+function handleDomLoaded(event) {
+  if (data.entries.length === 0) {
+    showNoEntry();
+  }
+  for (var i = 0; i < data.entries.length; i++) {
+    showNoEntry();
+    hideNoEntry();
```
Comment (codingcodymiller) [inline js/main.js:74]: What is the purpose of these lines?

File: js/main.js [hunk 4]
```diff
@@ -24,5 +24,98 @@ function submitForm(event) {
...
+
+var $anchor = document.querySelector('a');
+$anchor.addEventListener('click', anchor);
+
+function anchor(event) {
+  event.preventDefault();
+  var viewEntry = false;
+  var entryView = event.target.getAttribute('data-view');
+  switchView(entryView);
+  if (viewEntry === false) {
```
Comment (codingcodymiller) [inline js/main.js:106]: What is this conditional for?

File: js/main.js [hunk 5]
```diff
@@ -24,5 +24,98 @@ function submitForm(event) {
...
+function newButton(event) {
+  var newEntry = event.target.getAttribute('data-view');
+  switchView(newEntry);
+}
+
+var $anchor = document.querySelector('a');
+$anchor.addEventListener('click', anchor);
+
+function anchor(event) {
+  event.preventDefault();
```
Comment (codingcodymiller) [inline js/main.js:102]: This isn't a `submit` event handler, this line shouldn't be necessary

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by moamel97** [comment]
Kapture 2021-11-04 at 16 32 10

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**